scripts/mineAdditionalEntries.ts

```diff
@@ -0,0 +1,101 @@
+import * as anchor from '@coral-xyz/anchor';
+import { Program } from '@coral-xyz/anchor';
+import { DriftCompetitions } from '../ts/sdk/src/types/drift_competitions';
+import { CompetitionsClient } from '../ts/sdk/src';
+import { DriftClient, PublicKey, isVariant } from '@drift-labs/sdk';
+import dotenv from 'dotenv';
+import { getCompetitorAddressSync } from '../ts/sdk/src/addresses';
+import { program, Option } from 'commander';
+
+dotenv.config();
+
+function sleep(ms) {
+	return new Promise((resolve) => setTimeout(resolve, ms));
+}
+const ENV = 'mainnet-beta';
+
+const RPC_ENDPOINT =
+	process.env.RPC_OVERRIDE ?? 'https://api.' + ENV + '.solana.com';
+
+async function mineEntries(provider, authority: PublicKey, n = 10000) {
+	// Configure client to use the provider.
+	anchor.setProvider(provider);
+
+	const payer = (provider.wallet as anchor.Wallet).payer;
+	console.log(`PAYER: ${payer.publicKey}`);
+
+	const driftClient = new DriftClient({
+		connection: provider.connection,
+		env: ENV,
+		wallet: provider.wallet,
+		authority: authority,
+	});
+
+	const competitionClient = new CompetitionsClient({
+		driftClient,
+	});
+
+	const program = competitionClient.program as Program<DriftCompetitions>;
+
+	const name = 'sweepstakes';
+	const competitionKey = competitionClient.getCompetitionPublicKey(name);
+	let competitionAccount = await program.account.competition.fetch(
+		competitionKey
+	);
+
+	await competitionClient.driftClient.subscribe();
+	const details = await competitionClient.getCompetitionDetails(competitionKey);
+
+	console.log('max prize: ', details.prizePools[2].toNumber() / 1e6);
+
+	if (isVariant(competitionAccount.status, 'active')) {
+		const competitorKey = getCompetitorAddressSync(
+			competitionClient.program.programId,
+			competitionKey,
+			authority
+		);
+
+		const userStatsKey = driftClient.getUserStatsAccountPublicKey();
+
+		for (let i = 1; i <= n; i++) {
+			console.log('claiming! entry:', i);
+			try {
+				competitionClient.claimEntry(
+					competitionKey,
+					competitorKey,
+					userStatsKey
+				);
+				await sleep(1000);
```

Comment:
Wait, so this will run for 10k seconds if the default value for n is used?

Reply:
the default was overriden but updated so its required and default without --param is 5